### DISCUSSION TITLE
token连接失败

### DISCUSSION THREAD

**Discussion by szsyzx** [Qexo/Qexo#229] [General]
![image](https://user-images.githubusercontent.com/21995452/192728156-0016c2ef-2a8c-43a8-8467-512a1419dc70.png)

**Comment by am-abudu** [comment]
请查看文档 常见问题

**Reply by kekealiang** [reply]
我也是存在这个问题，一直提示“远程连接错误!请检查Token或分支是否正确”能否留个QQ或是微信指教一下，谢谢啦，常见问题里没找到答案，我是用vercel建立的HEXO,新建和以前建的都连接不上。

**Reply by am-abudu** [reply]
我的qq联系方式是2753534031

因为学业问题，我无法保证永远在线，如果有问题请留言

**Comment by kekealiang** [comment]
搞不好睡不着了




叶落
***@***.***



&nbsp;

**Comment by kekealiang** [comment]


**Reply by am-abudu** [reply]
你不应该把token截图在这里。你需要尽快将其注销。
你的问题是Github仓库里面不能带空格

**Comment by Jeamorg** [comment]
> ![image](https://user-images.githubusercontent.com/21995452/192728156-0016c2ef-2a8c-43a8-8467-512a1419dc70.png)

token 权限不够？
